README.md

```diff
@@ -83,5 +83,11 @@ let consumer = yield messageBus.consume.start(function (message, done) {
 yield consumer.stop();
 ``` 
 
+### userManagement
+- Allows to get user details object from auth0 API:
+```
+let userDetails = co(userManagement.getUserDetails(message.dataPayload.uuid)).catch(err => logger.error(err));
```

Comment:
``co`` should only be used in the top level of an application (or not at all if we are in koa context), so this should be - 
```
let userDetails = yield userManagement.getUserDetails(message.dataPayload.uuid);
```